sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GalleryApplicationVersion.cs

```diff
@@ -43,10 +43,11 @@ public GalleryApplicationVersion()
         /// <param name="provisioningState">Possible values include:
         /// 'Creating', 'Updating', 'Failed', 'Succeeded', 'Deleting',
         /// 'Migrating'</param>
-        public GalleryApplicationVersion(string location, GalleryApplicationVersionPublishingProfile publishingProfile, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string provisioningState = default(string), ReplicationStatus replicationStatus = default(ReplicationStatus))
+        public GalleryApplicationVersion(string location, GalleryApplicationVersionPublishingProfile publishingProfile, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), GalleryApplicationVersionSafetyProfile safetyProfile = default(GalleryApplicationVersionSafetyProfile), string provisioningState = default(string), ReplicationStatus replicationStatus = default(ReplicationStatus))
```

Comment:
This method will require a new file in the Customization folder, or at least a new method in an existing Customization file, as per https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/182581/Creating-Customization-Constructor